enclave-tls/samples/enclave-tls-client/echo.go

```diff
@@ -2,7 +2,7 @@ package main
 
 /*
 #cgo CFLAGS: -I../../src/include
-#cgo LDFLAGS: -L../../src -lenclave_tls -lm
+#cgo LDFLAGS: -L../../src -Wl,-rpath,'/opt/enclave-tls/lib' -lenclave_tls -lm
```

Comment:
Use runpath instead of rpath. In order to enable runpath instead of rpath, use `-Wl,-rpath,'/opt/enclave-tls/lib',--enable-new-dtags`.